.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,31 @@
+* [ ] **I have checked that this bug has not yet been reported by someone else.**
+* [ ] **I have checked that this bug appears on Chrome.**
+
+### Version
+ - [ ] latest
+ - [ ] other: please specify
+ 
+### Environment
+   - [ ] my computer
+       - [ ] Windows
+       - [ ] Linux
+       - [ ] macOS
+   - [ ] Cloud
+   - [ ] other: please specify
+
+### Actual behavior
+
+...
+
+### Steps to reproduce
```

Comment:
I would put "Steps to reproduce" first.

"Here's what I did. I expected the software to do this, but it did that other thing instead which I think is wrong."

Reply:
==>
### Steps to reproduce

...

### Expected behavior

...

### Actual behavior

...